modules/build/src/test/scala/scala/build/tests/DirectiveTests.scala

```diff
@@ -421,4 +426,46 @@ class DirectiveTests extends munit.FunSuite {
         }
     }
   }
+
+  test("dependency resolution errors with positions") {
+    val testInputs = TestInputs(
+      os.rel / "simple.sc" ->
+        """//> using dep org.xyz::foo:0.0.1
+          |//> using dep org.qwerty::bar:0.0.1
```

Comment:
Add correct dependency here to ensure the warning message, indicating the dependency can't be downloaded, is no longer displayed.